FreeRTOS/Demo/CORTEX_M3_MPS2_QEMU_GCC/main_full.c

```diff
@@ -910,36 +910,36 @@ static void prvDemonstrateChangingTimerReloadMode( void * pvParameters )
 {
     TimerHandle_t xTimer;
     const char * const pcTimerName = "TestTimer";
-    const TickType_t x100ms = pdMS_TO_TICKS( 100UL );
+    const TickType_t x50ms = pdMS_TO_TICKS( 50UL );
```

Comment:
why are we reducing the ticktime. It will be good to add the reasoning

Reply:
The duration of prvDemonstrateChangingTimerReloadMode is reduced from x100ms to x50ms to ensure that the IDLE task runs and reclaims prvDemonstrateChangingTimerReloadMode task’s resources before suicidal tests run. This will guarantee the accurate number of running tasks are returned in suicidal tests.

Reply:
I was suggesting to add this as a comment in the actual code at Line no 913.

Reply:
Got it, will update.